src/main/java/com/project/spring/digitalwallet/config/SecurityConfig.java

```diff
@@ -39,29 +38,26 @@
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-        http.csrf().disable()
-                .authorizeRequests()
-                .antMatchers(POST, "/api/login", "/api/register").permitAll()
-                .antMatchers(GET, "/api/register/user").hasAnyRole(ALL_ROLES)
-                .antMatchers(GET, "/api/users/**").hasAnyRole(ALL_ROLES)
-                .antMatchers(DELETE, "/api/users/**").hasAnyRole(ALL_ROLES)
-                .antMatchers(POST, "/api/send-money/**")
-                .hasAnyRole(Role.ADMIN.toString(), Role.USER.toString())
-                .antMatchers(POST, "/api/group/**")
-                .hasAnyRole(Role.ADMIN.toString(), Role.USER.toString())
-                .antMatchers(POST, "/api/transactions-history").hasAnyRole(ALL_ROLES)
-                .antMatchers(GET, "/api/transactions-history").hasAnyRole(ALL_ROLES)
-                .antMatchers(POST, "/api/send-money").hasAnyRole(ALL_ROLES)
-                .antMatchers(POST, "/api/upload").hasAnyRole(ALL_ROLES)
-                .antMatchers(GET, "/api/recurring-payments").hasAnyRole(ALL_ROLES)
-                .antMatchers(POST, "/api/recurring-payments").hasAnyRole(ALL_ROLES)
-                .antMatchers(DELETE, "/api/recurring-payments/**").hasAnyRole(ALL_ROLES)
-                .antMatchers(PUT, "/api/recurring-payments/**").hasAnyRole(ALL_ROLES)
-                .antMatchers(POST, "/api/payment-instruments/**").hasAnyRole(ALL_ROLES)
-                .antMatchers(PUT, "/api/payment-instruments/**").hasAnyRole(ALL_ROLES)
-                .antMatchers(GET, "/api/permissions").hasAnyRole(ALL_ROLES)
-                .and().exceptionHandling().authenticationEntryPoint(jwtAuthenticationEntryPoint)
-                .and().sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS);
+        http.csrf().disable().authorizeRequests().antMatchers(POST, "/api/login", "/api/register").permitAll()
+                .antMatchers(GET, "/api/register/user").hasAnyRole(ALL_ROLES).antMatchers(GET, "/api/users/management")
```

Comment:
The formatting is really bad and almost non-readable here but it will work i guess ...